inspirehep/modules/theme/jinja2filters.py

```diff
@@ -677,3 +677,28 @@ def format_date(datetext):
             return date
         elif len(datestruct) == 1:
             return datestruct[0]
+
+
+@blueprint.app_template_filter()
+def is_external_link(url):
+    """Checks if given url starts with the following."""
+    return not url.endswith(('.pdf', '.png', '.jpg', '.jpeg'))
+
+
+@blueprint.app_template_filter()
+def is_institute(institute):
+    """Checks if given url starts with the following."""
+    if institute.lower() in ['kekscan', 'ads', 'cds', 'hal', 'msnet', 'msnet']:
+        return True
+    return False
```

Comment:
This can just be `return institute.lower() in ['kekscan', 'ads', 'cds', 'hal', 'msnet', 'msnet']`.